python/pyspark/tests/test_worker.py

```diff
@@ -95,6 +95,14 @@ def raise_exception(_):
             self.assertRaises(Exception, lambda: rdd.foreach(raise_exception))
         self.assertEqual(100, rdd.map(str).count())
 
+    def test_after_non_exception_error(self):
```

Comment:
Can you add a short description with JIRA id? See "Pull Request" in http://spark.apache.org/contributing.html

Reply:
updated